blob.py

```diff
@@ -119,9 +120,19 @@ def download(context, backend, handle, output, format):
     type=click.Choice(["GENERIC", "RANDOM"], case_sensitive=False),
 )
 @click.option("--tempo", default=1.0, type=float)
+@click.option(
+    "--tracks",
+    default="AUTO",
+    type=str,
+)
+@click.option(
+    "--phoneme_fill_in",
+    default="SIL",
+    type=click.Choice(["SIL", "A", "E", "I", "O", "U"], case_sensitive=False)
+)
```

Comment:
I think that it would be nice to keep options short and memorable, using descriptions
```suggestion
@click.option(
    "--fill",
    default="SIL",
    type=click.Choice(["SIL", "A", "E", "I", "O", "U"], case_sensitive=False),
    help="Fill in phoneme for tracks without lyrics."
)
```